cmd/kops/validate_cluster.go

```diff
@@ -172,8 +172,8 @@ func RunValidateCluster(f *util.Factory, cmd *cobra.Command, args []string, out
 		// do we need to print which instance group is not ready?
 		// nodes are going to be a pain
 		fmt.Fprint(out, "\nValidation Failed\n")
-		fmt.Fprintf(out, "Master(s) Not Ready %d out of %d.\n", len(validationCluster.MastersNotReadyArray), validationCluster.MastersCount)
-		fmt.Fprintf(out, "Node(s) Not Ready   %d out of %d.\n", len(validationCluster.NodesNotReadyArray), validationCluster.NodesCount)
+		fmt.Fprintf(out, "Ready Master(s) %d out of %d.\n", len(validationCluster.MastersNotReadyArray), validationCluster.MastersCount)
+		fmt.Fprintf(out, "Ready Node(s) %d out of %d.\n", len(validationCluster.NodesNotReadyArray), validationCluster.NodesCount)
```

Comment:
Have not seen this in action, but should we show how many nodes are up, and how many are not?  
Also, we need to do a get component status, and pull all of the containers in kube-system.  Make sure everything is happy.  see https://github.com/kubernetes/kops/issues/1719 ... another PR